de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/DatabaseConfiguration.java

```diff
@@ -72,6 +72,17 @@ public DatabaseConfiguration(String databaseDriver, String jdbcURL, String host,
         this.setJdbcURL(jdbcURL);
     }
 
+    /**
+     * @return {@code True} if collation is supported by the database backend, else {@code false}.
+     */
+    boolean supportsCollation() {
+        if(databaseDriver!=null) {
+            return databaseDriver.contains("mysql");
```

Comment:
Is there the "risk" that somebody might be using MariaDB as an alternative to MySQL?

Reply:
@reckart  Could be, but - as of today - there is no "official" support for other DBMSs by JWPL. Other parts of the current code base would also be affected to fail with different `databaseDriver` values, i.e., many other things would also fail at runtime. 

I suggest you open a new issue, which has the topic to scan/re-work relevant code fragments to support MariaDB, as an enhancement. This way, it would be covered under a consistent enhancement change. Please link / quote the above code section into the issue description.